DigitalLearningSolutions.Data/Services/RegistrationService.cs

```diff
@@ -503,11 +500,13 @@ private void ValidateCentreEmail(int userId, string centreEmail)
             DelegateRegistrationModel delegateRegistrationModel
         )
         {
+            var delegateEmails = new List<string?>
+                { delegateRegistrationModel.PrimaryEmail, delegateRegistrationModel.CentreSpecificEmail };
+

```

Comment:
Unless I missed something, it looks like this is the only place which can trigger a call to `SupervisorDelegateDataService.GetPendingSupervisorDelegateRecordsByEmailsAndCentre` with an empty email in the list. Related to Simon's comment about having nulls in the email list, maybe it makes sense to only add to the list if the emails are not null here?